extension.json

```
@@ -2,7 +2,7 @@
   "$": "ime.extension.theme",
   "meta": {
     "id": "com.catppuccin.florisboard",
-    "version": "1.0.1",
+    "version": "2.0.0",
```

The version is automatically bumped in our CI/CD based on the conventional commit so no need to change it here

```suggestion
    "version": "1.0.1",
```